go/obscuronode/nodecommon/rpc/generated/README.md

```diff
@@ -7,6 +7,13 @@ Install Protobuf and Protoc-gen-go with:
 
     go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
 
+Add this to the bash file:
```

Comment:
To the bash file ? Do you mean the terminal env vars ?

Reply:
yes. I'll be more explicit